pkg/cmd/controller/pipelinerunner.go

```diff
@@ -168,9 +214,12 @@ func (o *PipelineRunnerOptions) startWorkers(ctx context.Context, wg *sync.WaitG
 		for {
 			select {
 			case <-ctx.Done():
-				logger.Info("Shutting down HTTP server")
+				logger.Info("shutting down HTTP server")
 				ctx, cancel := context.WithTimeout(ctx, shutdownTimeout*time.Second)
 				_ = srv.Shutdown(ctx)
+				if o.metaPipelineClient != nil {
+					_ = o.metaPipelineClient.Close()
```

Comment:
Maybe we should log this error in debug.

Reply:
makes sense